src/Main.java

```diff
@@ -0,0 +1,10 @@
+import java.io.File;
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
You know what to do)

![image](https://user-images.githubusercontent.com/27965136/147140919-5acbe9cc-b823-4ed5-924c-a6ef2bd74cd8.png)

Reply:
See sample from the previous project above.

Reply:
worked out an exception